editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
I recently published a Calypso-like memoization library: https://github.com/aduth/rememo

Curious if you have thoughts on whether `reselect` is a better approach or not.